Cargo.toml

```diff
@@ -56,7 +58,7 @@ widestring = "0.5.1"
 pyo3-build-config = { path = "pyo3-build-config", version = "0.17.1", features = ["resolve-config"] }
 
 [features]
-default = ["macros"]
+default = ["macros", "chrono"]
```

Comment:
When we get around to merging this we should remove it from the defaults here. Also mention in `features.md`.